library/src/main/java/com/schibsted/spain/barista/assertion/BaristaAssertions.kt

```diff
@@ -40,6 +40,11 @@ object BaristaAssertions {
         assertDisplayed(withText(text))
     }
 
+    @JvmStatic
+    fun assertDisplayed(@IdRes resId: Int, text: String) {
+        onView(resId.resourceMatcher()).check(matches(withText(text)));
```

Comment:
Watch out! Here you don't want to use our magic (I think). 
`resourceMatcher()` is intended to give you the matcher for an id **or** a string resource.

That means you can use `assertDisplayed(R.string.hello, "hello")`. I think it's best that we limit the matcher to `withId(id)`, so users won't get unexpected results if they use the API wrong.

Reply:
Uuuuuuh it's the **EDD** time!